contrib/kafka/OWNERS

```diff
@@ -0,0 +1,13 @@
+# These OWNERS are in addition to the repo level OWNERS.
```

Comment:
I can't find good documentation for this, but I think this is true based on the option being [NoParentOwners](https://github.com/kubernetes/test-infra/blob/d7f84dcb0ad73cbd7d3e8edde6abb2b7fa06bbb5/prow/repoowners/repoowners.go#L48).

Reply:
My understanding is that the comment is correct, subpath-owners are additive. You may have already seen this, but docs for the approval algorithm are at https://github.com/kubernetes/test-infra/tree/master/prow/plugins/approve/approvers.